TACC/Core-Styles
# feat(elements): monospace

## Overview / Changes

- Demo monospace elements.
- Tweak monospace styles.
- Isolate monospace styles.
- Add `:--monospace` custom selector.

## Related

- required by https://github.com/TACC/TACC-Docs/pull/2

## Testing

1. Open and review monospace demos:
    - http://localhost:3000/components/detail/monospace--cms
    - http://localhost:3000/components/detail/monospace--docs
2. Verify no difference.
3. Verify styles are similar to https://cep.tacc.utexas.edu/guides/data-transfer-guide/#sftp_transfer.

## UI

| demo | cms, full-screen | docs, full-screen |
| - | - | - |
| ![demo](https://user-images.githubusercontent.com/62723358/213839650-a7df7299-0cd0-40da-b708-551d595ed22b.png) | ![cms](https://user-images.githubusercontent.com/62723358/213839651-9bf0fa0e-df68-471e-9380-b1b8a2186e14.png) | ![docs](https://user-images.githubusercontent.com/62723358/213839652-2477376b-db5a-4254-9cad-f11ce64ef7b4.png) |
